packages/stores/src/query/agoric/vbank-assets.ts

```diff
@@ -0,0 +1,39 @@
+import { ChainGetter } from "../../chain";
+import { makeObservable } from "mobx";
+import { QuerySharedContext } from "../../common";
+import { ObservableChainQueryRPC } from "../chain-rpc-query";
+import { VstorageResult } from "./types";
+
+export class ObservableQueryVbankAssets extends ObservableChainQueryRPC<VstorageResult> {
+  constructor(
+    sharedContext: QuerySharedContext,
+    chainId: string,
+    chainGetter: ChainGetter
+  ) {
+    super(
+      sharedContext,
+      chainId,
+      chainGetter,
+      '/abci_query?path="custom/vstorage/data/published.agoricNames.vbankAsset"'
```

Comment:
Is there no way to query this through the rest API? Query via rpc is not preferred at all

Reply:
Unfortunately not, and I just realized, when testing on mainnet, Keplr's RPC node for Agoric does not support this query.

It returns a 404 when I fetch: https://rpc-agoric.keplr.app/abci_query?path=%22custom/vstorage/data/published.agoricNames.vbankAsset%22

However, this other node (on mainnet) returns the data correctly: https://agoric-rpc.polkachu.com/abci_query?path=%22custom/vstorage/data/published.agoricNames.vbankAsset%22

What would it take to enable this for Keplr's node?

Reply:
Oh wait, it looks like there is a RESTful way to fetch this data: https://main.api.agoric.net/agoric/vstorage/data/published.agoricNames.vbankAsset

However, similar issue with Keplr's node giving a 404 https://lcd-agoric.keplr.app/agoric/vstorage/data/published.agoricNames.vbankAsset

Reply:
It's open now. https://lcd-agoric.keplr.app/agoric/vstorage/data/published.agoricNames.vbankAsset
We are whitelisting the endpoints because of protection purposes against strange requests.